frontend/src/main/scala/bloop/bsp/BloopBspServices.scala

```diff
@@ -863,6 +864,20 @@ final class BloopBspServices(
   private def toBuildTargetId(project: Project): bsp.BuildTargetIdentifier =
     bsp.BuildTargetIdentifier(project.bspUri)
 
+  private def toJvmBuildTarget(project: Project): bsp.JvmBuildTarget = {
+    Option(project.platform)
```

Comment:
We have some of this logic already in a method called `project.jdkConfig`. What do you think about using that instead of this? We can also replace the values of `java.home` and the java version and use those that are available in `JavaRuntime`.

Reply:
yeah, I guess it is enough to use project.jdkConfig, anyway it sets the correct jvm default config if not provided, and if it returns none, JvmBuildTarget can be empty as it means native or js.